composer.json

```diff
@@ -11,19 +11,19 @@
         }
     ],
     "require": {
-        "php": "^7.3|^8",
+        "php": "^7.3|^8.0",
         "ext-simplexml": "*",
-        "illuminate/config": "^6.0 || ^8.0",
```

Comment:
i think we should still support laravel 8, still has over a year of bug fix support from laravel. Shouldn't this then be `"illuminate/config": "^6.0 || ^8.0 || ^9.0",`?

Reply:
@mr-feek laravel added native template-params for Collections. 
I think it will be easier to split two tags: `^1.0` for laravel 6 | 8 and `^2.0` for `^9.0`.

I want to believe that this plugin will be unnecessary in the nearest future because of full native support (e.g. Builder).

Reply:
> I want to believe that this plugin will be unnecessary in the nearest future because of full native support (e.g. Builder).

Unfortunately, a dedicated plugin will be necessary as long as Laravel doesn't annotate conditional types for "overloaded" methods like `str()` (new) and `Request::validated()` (changed in L9), which return a different type based on the number of parameters provided.